.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "bashtester"]
+	path = bashtester
+	url = ../bashtester
```

Comment:
Shouldn't this be a URL?

Reply:
It can also be a relative path on the repo since both are on the same host.

Reply:
K